wallet/apps/dapps.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
Can we put more dummy-content for now, given we need to fill this in later?

Reply:
Yes we can.